databaseCreation.sql

```diff
@@ -43,4 +45,15 @@ CREATE TABLE `WhiteListedRoles` (
   `guildId` VARCHAR(255) NOT NULL,
   `id` VARCHAR(255) NOT NULL,
   PRIMARY KEY (`guildId`, `id`)
+) ENGINE=InnoDB DEFAULT CHARSET=latin1;
+
+DROP TABLE IF EXISTS `Muted`;
+CREATE TABLE `Muted` (
+  `guildId` VARCHAR(255) NOT NULL,
+  `userId` varchar(255) NOT NULL,
+  `byUserId` varchar(255) DEFAULT NULL,
+  `date` datetime DEFAULT NULL,
+  `until` datetime DEFAULT NULL,
+  `reason` longtext,
+  PRIMARY KEY (`guildId`, `userId`)
```

Comment:
This will throw an error if a user has been muted a second time

Reply:
It shouldn't as every time someone it muted, it checks if this person is already muted and removes this entry from the database for the new mute. There was however a separate little bug in this, so it's better to have checked anyway.